client/src/app/shared/components/FilterToolbar/FilterToolbar.tsx

```diff
@@ -75,6 +77,7 @@ export const getFilterLogicOperator = <
 
 export interface IFilterToolbarProps<TItem, TFilterCategoryKey extends string> {
   filterCategories: FilterCategory<TItem, TFilterCategoryKey>[];
+  filterGroups: string[];
```

Comment:
You might be able to totally get away with not having a `filterGroups` param here at all. All the filterCategories have their `filterGroup`, so the `filterGroups` could be derived from those below.